install_config/install/docker_registry.adoc

```diff
@@ -72,6 +72,21 @@ This creates a service and a deployment configuration, both called
 *docker-registry*. Once deployed successfully, a pod is created with a name
 similar to *docker-registry-1-cpty9*.
 
+Use `-selector` to deploy the router to any node(s) that match a specified node label:
```

Comment:
Here "the router" seem like a copy/paste error. And `-selector` should be correct to `--selector`.